Author: Daniel Kahn Gillmor <dkg@fifthhorseman.net>

The only thing the test suite relies on from parking_lot is
parking_lot::Mutex, which should be identical in use even in 0.11

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -24,8 +24,7 @@
 
 [dependencies]
 [dev-dependencies.parking_lot]
-version = "0.5"
-features = ["nightly"]
+version = "0.11"
 [badges.travis-ci]
 branch = "master"
 repository = "SergioBenitez/yansi"
